lib/libmage6.py

```diff
@@ -0,0 +1,155 @@
+# coding: utf-8
+
+'''
+    Python MAGE client lib for python scripts
+    @license: Apache License, Version 2.0
+    @copyright: 2015 Sebastien Renard
+    @author: Sebastien Renard
+'''
+
+
+from urlparse import urljoin
+import logging
+from logging.handlers import RotatingFileHandler
+
+import requests
+
+
+class MageClient(object):
+    """Mage client that connect to Mage server though HTTP(S) REST web services"""
+
+    def __init__(self, base_url="http://127.0.0.1:8081", username= "root", password = "secret", setup_log=True):
```

Comment:
Could you please remove the default values for login and password, and raise an error if not set? I believe having default values would only confuse users.